warehouse/models/mart/audit/fct_bigquery_data_access.sql

```diff
@@ -0,0 +1,33 @@
+{{ config(materialized='table') }}
+
+WITH audit_fct_bigquery_data_access AS (
+    SELECT
+        timestamp,
+        date,
+        severity,
+        resource_name,
+        principal_email,
+        job_name,
+        job_type,
+        dbt_invocation_id,
+        create_disposition,
+        destination_table,
+        priority,
+        query,
+        statement_type,
+        write_disposition,
+        duration_in_seconds,
+        referenced_tables,
```

Comment:
can you either unnested in this model or create another table on top that unnests? I think seeing a rough breakdown of data volumes by queried table (i.e. read not write) may be useful; a table that is expensive to build but efficient to query is probably preferable for most of our uses cases.